UserRelated/Main.py

```diff
@@ -0,0 +1,208 @@
+"""
+This module get a list of preferences from the user as command-line arguments and finds similar osm-tracks according to
+the request.
+
+Command-Line Arguments Example:
+baiersbronn 48.6 48.5 8.4 8.3 0 0 0 0 0 1 0 1 1 1 2
+
+Note:
+Pay attention that we don't have a real osm-DB yet, so in order to run this module, please generate a fake DB first
+with createExampData.
+"""
+
+
+import argparse
+import json
+from UserRelated import ResultsTests as tests
+from datasketch import MinHash, MinHashLSH
```

Comment:
i saw that mingashlsh forest supports top k queries: https://pypi.org/project/datasketch/ 
i didn't get it all the way through so im putting it here for you to reflect upon

Reply:
I'ts a question of what service we want to give to the user. If we stay with the current implementation, we can guarantee that the resulted tracks will always be similar to the user request (but on the other side, we might not give her/him any result if there isn't a good match). If we choose to exchange the implementation to top-k, we will always give the user something, it just might not be a very good match, if at all.